### ISSUE TITLE
options.variants not cannot be returned

### ISSUE THREAD

**Issue by amlcodes** [tinloof/sanity-kit#23]
How are options.variants supposed to be used? Right now it's omitted from the schema type per the typescript definitions AFIAK. I'd expect the option selection to be queryable and be used in the frontend.

**Comment by Jamiewarb** [comment]
As far as I know, `options.variants` is only used to set up the Sections functionality, and is used in the Sections picker. I don't imagine the Frontend would need any information from `options.variants`. Traditionally the `schemaTypes` are for setting up the CMS

It may be helpful to answer this issue by providing more context around what you're trying to do, and doubly so if you're able to set up a small reproduction

**Comment by amlcodes** [comment]
right as i said, it's omitted from the schematype actually passed to sanity, the context is that the Options.variants takes an array of options including variant name and an asset image. This is either misnamed with useless arguments or missing functionality and usefulness that would be expected. As it stands now, there is no usefulness for multiple variants as an array. If you set up an additional variant and choose it, there is no additional context being returned from a quer, or any additional information being stored in the sanity data lake

**Comment by Jamiewarb** [comment]
Sanity itself does not use `options.variants`. It's used by the custom components to display the variants in the Sections selector. That part is working as expected.

What is it you're trying to do?

**Comment by amlcodes** [comment]
@Jamiewarb yes exactly! ahhh that's how it's done, I was trying to use reference lol. Thank you mate!
